keps/sig-storage/2485-read-write-once-pod-pv-access-mode/README.md

```diff
@@ -1086,6 +1086,24 @@ An operator can query for PersistentVolumeClaims and PersistentVolumes in the
 cluster with the ReadWriteOncePod access mode. If any exist then the feature is
 in use.
 
+###### How can someone using this feature know that it is working for their instance?
+
+<!--
+For instance, if this is a pod-related feature, it should be possible to determine if the feature is functioning properly
+for each individual pod.
+Pick one more of these and delete the rest.
+Please describe all items visible to end users below with sufficient detail so that they can verify correct enablement
+and operation of this feature.
+Recall that end users cannot usually observe component logs or access metrics.
+-->
+
+- [X] Other
+  - Details:
+    - Create a Pod using a PersistentVolumeClaim with the ReadWriteOncePod access mode
+    - A PersistentVolume should be created and the PersistentVolumeClaim .status.phase=Bound
+    - A VolumeAttachment should be created with .status.attached=True
```

Comment:
PVs and VolumeAttachments are cluster-scoped, so a namespace editor won't be able to see them.  Are there any events or pod conditions that are set?

Reply:
Included events and status conditions and prefixed checking the cluster-scoped assets if access available